### PR TITLE
Restaurant categories

### PR DESCRIPTION
**Pull request by lcegit**
## PT Link: https://www.pivotaltracker.com/story/show/154205898

### CODE REVIEW

**Comment by lcegit** [comment]
@aljoh @oliverochman Please review ^

File: features/restaurant_categories.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      | Thomas Kebabrulle  | Chinese  |
+      | Oliver's Burger    | Chinese  |
+      | Borat's Palace     | French   |
+      | Lisa's Bistro      | French   |
+      | Franchesco's Pasta | Italian  |
+      | Mama's Pizza       | Italian  |
+
+  Scenario:
+    When I visit the landing page
+    Then show me the page
```
Comment (aljoh) [inline features/restaurant_categories.feature]: Remove "show me the page"

**Comment by lcegit** [comment]
There's a previous factory for creating restaurants, it could possibly be omitted here.

**Review by lcegit — COMMENTED** [review]
The merge conflict was solved manually.

File: app/views/restaurant/index.html.erb [hunk 2]
```diff
@@ -1,9 +1,14 @@
...
+<h1>Dishley House</h1>
 <p>What do you feel like tonight?</p>
 
-<% @restaurants.each do |restaurant| %>
-  <div id="restaurant-<%= restaurant.id %>">
-    <%= "#{restaurant.name}" %>
-    <%= link_to 'Show page', restaurant_path(restaurant) %>
+<% @restaurant_categories.each do |restaurant_category| %>
+  <div id="restaurant_category-<%= restaurant_category.name %>">
+    <%= "#{restaurant_category.name}" %>
```
Comment (tochman) [inline app/views/restaurant/index.html.erb]: Why do you string interpolate here?

File: app/views/restaurant/index.html.erb [hunk 3]
```diff
@@ -1,9 +1,14 @@
...
+    <% restaurant_category.restaurants.each do |restaurant| %>
+      <div id="restaurant-<%= restaurant.id %>">
+        <%= "#{restaurant.name}" %>
+        <%= link_to 'Show page', restaurant_path(restaurant) %>
```
Comment (tochman) [inline app/views/restaurant/index.html.erb]: Please use other wording, like 'Visit' or make the restaurant name a link.

**Review by tochman — CHANGES_REQUESTED** [review]
Close, but not quite there yet...

**Pull request merged**